app/src/main/java/co/eventbox/event/view/activities/MainActivity.kt

```diff
@@ -44,14 +48,17 @@ class MainActivity : AppCompatActivity() {
 
     }
 
-    fun locate(locate: String) {
-        val locale = Locale(locate)
-        Locale.setDefault(locale)
-        val resources: Resources = resources
-        val configuration: Configuration = resources.getConfiguration()
-        configuration.locale = locale
-        resources.updateConfiguration(configuration, resources.getDisplayMetrics())
-    }
+      fun locate(locate: String) {
```

Comment:
You can create a base Activity and insert code that might be used in another Activity

Reply:
you right, fixed 👍